src/Theming/colorDefinitions/blue.ts

```diff
@@ -0,0 +1,31 @@
+export const BLUE_ENUM = {
+  BLUE: 'blue',
+  BLUE50: 'blue50',
+  BLUE100: 'blue100',
+  BLUE200: 'blue200',
+  BLUE300: 'blue300',
+  BLUE400: 'blue400',
+  BLUE500: 'blue500',
+  BLUE600: 'blue600',
+  BLUE800: 'blue800',
+  BLUE900: 'blue900',
+
+  DARK_BLUE: 'dark_blue',
```

Comment:
According to DS2, these variants dark_* and light_* and older naming might be deprecated, am I right?